plugins/aws_sts/README.md

```diff
@@ -0,0 +1,14 @@
+# AWS STS Plugin
+
+This plugin allows integration between Samson and [AWS STS](https://docs.aws.amazon.com/STS/latest/APIReference/Welcome.html).
+
+## Overview
+
+Allows Samson to inject temporary security credentials for an AWS Role. The plugin uses
```

Comment:
What are the credentials that Samson is using to authenticate, and then assume a different role? Is the assumption that ENV vars are set in the Samson process itself? And that Samson has permission to assume into the role specified?

Reply:
@jonmoter Samson will use it's own AWS_ACCESS_KEY/SECRET to assume the role referenced by the ARN. The assumption is that Samson _should_ have permission to assume the role.

Reply:
add a note here that `Samson has to be authenticated to aws for this to work, for example AWS_ACCESS_KEY/SECRET`

Reply:
How do we know what IAM user Samson authenticates as? Is that shown in the UI somewhere?

Reply:
first sentence is repeat of the title ... can be removed ?